### DISCUSSION TITLE
2.8.1 overall test - how to merge all 2.8.1 milestone tagged PRs automatically?

### DISCUSSION THREAD

**Discussion by ParkerEde** [EdgeTX/edgetx#2980] [Q&A]
Currently I can build the firmware and a companion based on the TAG v2.8.0. What I can't get automated with GIT is to locally merge all milestone tagged 2.8.1 pullrequests into the TAG v2.8.0. Is there a convenient way, or do I have to manually merge every single 2.8.1 milestone pullrequest locally to be able to run a 2.8.1 overall test? Would appreciate any tips here.

**Comment by pfeerick** [comment]
The 2.8 branch should have most of the PRs for 2.8.1 already, they get added to that shortly after being merged into main. So generally you would simply use the current head build of 2.8 to get a preview of what 2.8.1 (and later) will look like... (just as `main` is the work in progress towards the next release).

**Reply by ParkerEde** [reply]
Okay thanks. I have done it like this now. 
git fetch --all
git checkout 2.8 and build simulator lib for Horus X12S.
But there are PRs (e.g. #2895 that were merged 2 weeks ago or #2969 3 days ago), they don't seem to be included in branch 2.8 yet. Can this be? Or am I making another mistake?

I'm on:
$ git rev-parse HEAD
4eda6e1e4661aae9f78d5780f4dcfafb2c944753
